Realm/RLMEmailPasswordAuth.h

```diff
@@ -41,9 +41,9 @@ NS_ASSUME_NONNULL_BEGIN
  @param completionHandler A callback to be invoked once the call is complete.
 */
 
-- (void)registerEmail:(NSString *)email
-             password:(NSString *)password
-           completion:(RLMEmailPasswordAuthOptionalErrorBlock)completionHandler;
+- (void)registerUserWithEmail:(NSString *)email
```

Comment:
This function needs `NS_SWIFT_NAME(registerUser(email:password:completion:)` so that it gets bridged into swift sensibly (rather than the default of having the first label be `withEmail:`).